src/sphsinglepcqueue.cpp

```diff
@@ -1232,14 +1232,14 @@ SPHSinglePCQueueSetCachePrefetch (SPHSinglePCQueue_t queue)
 	  touchptr += pagesz;
 	  touch += *touchptr;
 #if 0
```

Comment:
I know you aren't touching this, but why #if 0?
Why not #ifdef PRINT_DEBUG or #ifdef __SASDebugPrint__?